src/flamenco/runtime/program/fd_compute_budget_program.c

```diff
@@ -63,7 +63,10 @@ int fd_executor_compute_budget_program_execute_instructions( fd_exec_txn_ctx_t *
     if ( ret ) {
       FD_LOG_WARNING(("fd_compute_budget_program_instruction_decode failed"));
       FD_LOG_HEXDUMP_WARNING(("cbi data", data, instr->data_sz));
-      return FD_EXECUTOR_INSTR_ERR_INVALID_ACC_DATA;
+      ctx->instr_err_idx = i;
+      ctx->exec_err_kind = FD_EXECUTOR_ERR_KIND_INSTR;
+      ctx->exec_err = FD_EXECUTOR_INSTR_ERR_INVALID_ACC_DATA;
```

Comment:
Let's make macros like `FD_VM_ERR_FOR_LOG_INSTR`, so we can tweak the implementation without rewriting all programs? (same below)